app/src/main/java/com/ggetters/app/ui/central/views/NotificationsActivity.kt

```
@@ -1,55 +1,28 @@
-// TODO: Backend - Add notification grouping and smart categorization
-// TODO: Backend - Implement notification search and filtering
-// TODO: Backend - Add notification export and backup functionality
 
 @AndroidEntryPoint
 class NotificationsActivity : AppCompatActivity() {
 
     private val model: NotificationsViewModel by viewModels()
-    private lateinit var notificationAdapter: NotificationAdapter
-    private var allNotifications: List<NotificationItem> = emptyList()
-    private var filteredNotifications: List<NotificationItem> = emptyList()
+    private lateinit var notificationAdapter: NotificationCardAdapter
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)